t5x/train.py

```diff
@@ -703,13 +703,19 @@ def _main(argv: Sequence[str]):
 
     # check if all the required info is present for multiprocess runs
     if FLAGS.multiprocess:
-      err_msg = ('Provide values for `num_processes` and `process_id` variables')
-      assert FLAGS.coordinator_address != None and \
-             FLAGS.num_processes != None and \
-             FLAGS.process_id != None, err_msg
 
-      logging.info('Server address: %s, Total hosts: %d, Host ID: %d',
-                   FLAGS.coordinator_address, FLAGS.num_processes, FLAGS.process_id)
+      logging.info('Coordinator address: %s, Total Processes: %d, '
+                    'Process ID: %d',
+                   FLAGS.coordinator_address, FLAGS.num_processes,
+                   FLAGS.process_id)
+
+      err_msg = ('To properly use multiprocess, provide valid values for '
```

Comment:
```
if (FLAGS.coordinator_address == None or 
    FLAGS.num_processes == None or 
    FLAGS.process_id == None):
  raise ValueError(
      '`coordinator_address`, `num_processes` and `process_id` '
      'should not be None. Please provide those values to initialize the '
      'distributed system')
```

Please format it according to google's python style guide and the line limit is 80 characters.

Reply:
Still pending?

Reply:
I started a review apparently so my last few comments ended up in `pending` state. Anyways, my bad that I didn't realize that you suggested some code above. Thanks! I've now made the necessary changes.

I've also accordingly fixed formatting in the other PR:#608. Could you pls take a look at that? Thanks!